examples/src/Auction/Offchain.hs

```diff
@@ -148,8 +159,9 @@ txHammer offerUtxo =
             A.auctionValidator
             (\_ x -> x `Value.geq` theNft)
         (A.Offer seller _minBid) <- spOutGetDatum @A.Auction offerUtxo
+        -- checkWallet seller sellerWallet
```

Comment:
This seems to be the reason this tx fails BTW: the `seller` that's put into the required signers `TxSkel` field and the `sellerWallet` that this transaction is signed with (with `` `as` `` before and explicitly now) are two different wallets, hence the failure. I guess it's kind of expected, but does it really test what we want to test?